### ISSUE TITLE
[FEATURE] Side menu in careers page doesn't close when the user clicks outside the menu.

### ISSUE THREAD

**Issue by Timonwa** [Timonwa/techroadmap#78]
**Is your feature request related to a problem? Please describe.**
When a user opens the side menu on the careers page, if they click on other parts of the page that is not the side menu, the side menu should close.

**Comment by dangvu0502** [comment]
Hi Timonwa,
Is this what you mean?

https://user-images.githubusercontent.com/57172125/188319833-fca1bddb-6575-4f50-b992-81849532b804.mp4

**Comment by Timonwa** [comment]
Yes @dangvu0502 . This was what I meant. Thank you for wanting to implement it.
